project - test2/SPS-basics-11-5/script.js

```diff
@@ -0,0 +1,72 @@
+var userWin = 0;
+var userAttempt = 0;
+var userWinningPercentage = 0;
+
+var main = function (input) {
+  var randomSPS = randomSPSoutput();
+  console.log("random guess generated by Computer:");
+  console.log(randomSPS);
+
+  // Its a draw,
+  // IF input == random output
+
+  if (input == randomSPS) {
+    userAttempt += 1;
+    userWinningPercentage = (userWin / userAttempt) * 100;
+    userWinningPercentage = userWinningPercentage.toFixed(1);
+    return `It's a tie! <br> <br> Your guess: ${input}; <br> Computer guess: ${randomSPS} <br><br> You have attempted ${userAttempt} times and won ${userWin} times (${userWinningPercentage}%) ! <br><br> Type "scissors", "paper" or "stone" to play another round!`;
+  }
+
+  // You win,
+  // IF input == scissors && random output == paper
+  // OR input == paper && random output == rock
+  // OR input == rock && random output == scissors
+
+  if (
+    (input == "scissors" && randomSPS == "paper") ||
+    (input == "paper" && randomSPS == "stone") ||
+    (input == "stone" && randomSPS == "scissors")
+  ) {
+    userAttempt += 1;
+    userWin += 1;
+    userWinningPercentage = (userWin / userAttempt) * 100;
+    userWinningPercentage = userWinningPercentage.toFixed(1);

```

Comment:
could be refactored into a function that takes in userAttempt and userWin to return winning percentage
as this code is reused for draw, win and loss conditions